tst/EnergyPlus/unit/DaylightingManager.unit.cc

```diff
@@ -692,6 +867,18 @@ TEST_F(EnergyPlusFixture, DaylightingManager_GetDaylParamInGeoTrans_Test)
     EXPECT_NEAR(3.048, ZoneDaylight(1).DaylRefPtAbsCoord(1, 1), 0.001);
     EXPECT_NEAR(-2.048, ZoneDaylight(1).DaylRefPtAbsCoord(2, 1), 0.001);
     EXPECT_NEAR(0.9, ZoneDaylight(1).DaylRefPtAbsCoord(3, 1), 0.001);
+
+    DataGlobals::BeginSimFlag = true;
+    DataGlobals::WeightNow = 1.0;
+    DataGlobals::WeightPreviousHour = 0.0;
+    CalcDayltgCoefficients();
+    int zoneNum = 1;
+    // test that tmp arrays are allocated to correct dimension
+    // zone 1 has only 1 daylighting reference point
+    DayltgInteriorIllum(zoneNum);
+    zoneNum += 1;
+    // zone 2 has 2 daylighting reference points and will crash if not dimensioned appropriately.
+    DayltgInteriorIllum(zoneNum);
```

Comment:
It's great that you just attached to an existing unit test!  Since the test file takes so long, it is awesome to have a unit test to use as a validation source here.